lib/run-compilation.js

```diff
@@ -46,7 +46,11 @@ module.exports = function runCompilation (options, compiler, done) {
     }
 
     if (errors.length) {
-      compilation.errors.push(chalk.red(options.formatter(errors)));
+      if (options.emitErrors === false) {
+        compilation.warnings.push(chalk.yellow(options.formatter(errors)));
```

Comment:
What I was thinking was more like a text header or something, but the colours also make sense. Probably a combination to indicate that the errors (in red) are being "ignored" but still reported.